app/models/spree/adyen_common.rb

```diff
@@ -141,14 +140,6 @@ def set_up_contract(source, card, user, shopper_ip)
           response = authorize_on_card 0, source, options, card, { recurring: true }
 
           if response.success?
-            last_digits = response.additional_data["cardSummary"]
-            if last_digits.blank? && payment_profiles_supported?
-              note = "Payment was authorized but could not fetch last digits.
-                      Please request last digits to be sent back to support payment profiles"
-              raise Adyen::MissingCardSummaryError, note
-            end
-
-            source.last_digits = last_digits
```

Comment:
This isn't relevant to us as we're using client side encryption, we won't get the last digits back to save anyway